src/RestfulManager.php

```diff
@@ -183,6 +184,21 @@ public function process() {
   }
 
   /**
+   * Helper function to echo static strings.
+   *
+   * @param DataInterpreterInterface $value
+   *   The resource value.
+   * @param string $message
+   *   The string to relay.
+   *
+   * @return string
+   *   Returns $message
+   */
+  public static function echoMessage(DataInterpreterInterface $value, $message) {
```

Comment:
I'd rather keep this, since this is the name we have in 1.x. Also, it's not printing anything, just returning the passed in value.

Actually, `$message` should be set to `mixed` instead of `string`.